api/lib/domain/models/SmartPlacementAssessment.js

```diff
@@ -25,6 +27,8 @@ class SmartPlacementAssessment {
     this.id = id;
     // attributes
     this.state = state;
+    this.createdAt = createdAt;
+    this.type = type;
```

Comment:
get type{
return 'SMART_PLACEMENT';
}

Minimum : this.type = 'SMART_PLACEMENT';